capnp/lib/capnp.pyx

```diff
@@ -317,7 +306,7 @@ ctypedef fused PromiseTypes:
     _Promise
     _RemotePromise
     _VoidPromise
-    PromiseFulfillerPair
+    # PromiseFulfillerPair
```

Comment:
Needed?

Reply:
Yeah, the PromiseFulfillerPair does not at all support the same operations as the other classes in this fused type. So I had to remove it. (But I don't believe anything is lost by that.)